test/test_conversation_v1.py

```diff
@@ -752,6 +753,8 @@ def test_message():
 
     assert message is not None
     assert responses.calls[0].request.url == message_url1
+    assert 'x-watson-learning-opt-out' in responses.calls[0].request.headers
+    assert responses.calls[0].request.headers['x-watson-learning-opt-out'] == 'true'
```

Comment:
Nice check on the response. More confidence-inspiring than the previous `assert inited.x_watson_learning_opt_out`.